apps/DroidCam/install

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+#check for 32 bit arch with 64 bit kernel
+if [ "$(uname -m)" == "arm64" ] || [ "$(uname -m)" == "aarch64" ]; then
+  if  [ "$(dpkg --print-architecture)" == "armhf" ]; then
+    warning "You can't install DroidCam on armhf OS with arm64 kernel."
```

Comment:
I try my best to make arm64-kernel-on-armhf-OS an officially supported operating mode of Pi-Apps. Other apps like WPS Office need the arm64 kernel enabled, so it would be unfortunate for other apps to require the 64-bit kernel to *not* be enabled.

It should be possible to manually specify the desired architecture to those `make` commands.

Reply:
Previously, I tried many times for this, all failed.

Tried:
- Installing arm64 kernel 
  - Result: I got my kernel messed up and I need to reflash my SD.
- Change `make` to compile on armv7l kernel
  - Result: `make` prompts about a whole list of errors.
- Use `setarch` and cross-compilation 
  - Result: Same errors with compiling natively

Opened [an issue](https://github.com/dev47apps/droidcam/issues/218 on droidcam repo.